data/ut/legislature/Robert-M-Spendlove-11e28ca5-f9ab-452e-a922-7e757a0f880b.yml

```diff
@@ -0,0 +1,46 @@
+id: ocd-person/11e28ca5-f9ab-452e-a922-7e757a0f880b
+name: Robert M. Spendlove
+email: rspendlove@le.utah.gov
+image: https://le.utah.gov/images/legislator/SPENDRM.jpg
+party:
+- name: Republican
+roles:
+- start_date: 2021-01-01
+  type: lower
+  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
+  district: '42'
+offices:
+- classification: district
+  address: 2492 E BARCELONA DR, SANDY, UT, 84093
+links:
+- url: https://le.utah.gov/asp/billsintro/RepResults.asp?Listbox3=SPENDRM
+  note: legislation
+- url: https://le.utah.gov/Documents/demographic/profiles/2022/House_Dist42.pdf
```

Comment:
@sroomf this person should already have a file, we need to add the new role info:
- start_date: 2023-01-01
  type: lower
  jurisdiction: ocd-jurisdiction/country:us/state:ut/government
  district: '42'

Previously they served in house district 49 from jan 2015-Jan 2023